modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/eventTypes.js

```diff
@@ -12,10 +12,15 @@
  * details.
  */
 
-const SIDEBAR = {
+export const SIDEBAR = {
 	FIELD: {BLUR: 'sidebar_field_blur'},
+	FIELDSET: {
+		UPDATE: 'sidebar_fieldset_update',
+	},
 };
 
 export const EVENT_TYPES = {
 	SIDEBAR,
 };
+
+export default EVENT_TYPES;
```

Comment:
We usually have some rules regarding the named and default export in the same file in Infra, but just bringing it here, it would be good to just keep the export named because it makes people use the name of the variable that we want to spread across in the codebase to be consistent in the entire code base.